bridgenode/offsetfile.go

```diff
@@ -48,14 +48,18 @@ func buildOffsetFile(dataDir string, tip util.Hash,
 		}
 	}
 
-	lvdb, err := OpenIndexFile(dataDir)
-	if err != nil {
-		panic(err)
-	}
-	defer lvdb.Close()
+	lvdb := OpenIndexFile(dataDir)
```

Comment:
Doesn't catch the error...
actually I think I can merge anyway and just put it in